README.md

```diff
@@ -4,6 +4,11 @@
 
   <h1><b>Harshika Govind</b></h1>
   <h3><b>Portfolio Project Desktop Version</b></h3>
+  </div>
+  <div align="left">
+<h2 ><b>Demo Link</h2></b>
+<hr>
+https://harshi0102.github.io/Portfolio-Microverse-project/
 
 </div>
```

Comment:
- [optional] Kindly remove this lines of code, it occurs as if you're repeating what you already have in the LIVE DEMO header. 👍